nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -311,4 +318,23 @@ config PSA_CRYPTO_DRIVER_RSA_KEY_SIZE_1024_OBERON
 	  RSA with 1024 bit keys is not recommended for new designs.
 	  Please see https://www.keylength.com/
 
+config PSA_CRYPTO_DRIVER_ALG_JPAKE_OBERON
+	bool
+	prompt "PSA J-PAKE support - oberon" if !PSA_PROMPTLESS
+	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_JPAKE
+	depends on PSA_CORE_OBERON
+
+config PSA_CRYPTO_DRIVER_ALG_SPAKE2P_OBERON
+	bool
+	prompt "PSA SPAKE2+ support - oberon" if !PSA_PROMPTLESS
+	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_SPAKE2P
+	depends on PSA_CORE_OBERON
+
+config PSA_CRYPTO_DRIVER_ALG_SRP_OBERON
+	bool
+	prompt "PSA SRP support - oberon" if !PSA_PROMPTLESS
+	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_SRP
+	select EXPERIMENTAL
```

Comment:
Is the SRP the only experimental? Isn't it the same for jpake  and spake etc?

Reply:
I'll add it for the others